### PR TITLE
Consolidate Black and isort configs

### PR DESCRIPTION
**Pull request by kaxil**
# Description
## What is the current behavior?

For some days now there have been a lot of conflicts between `isort` and `black` when run via pre-commit hook on local vs CI (pre-commit.ci).

Example:
- https://github.com/astronomer/astro-sdk/pull/1024
- https://github.com/astronomer/astro-sdk/pull/1001
- https://github.com/astronomer/astro-sdk/pull/994
- https://github.com/astronomer/astro-sdk/pull/990

## What is the new behavior?

This PR fixes the conflicts and uses https://pycqa.github.io/isort/docs/configuration/black_compatibility.html and https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#isort .

The default line-length for black is 88 ([reference](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html?highlight=length#line-length))

Whereas it is 79 for isort ([reference](https://pycqa.github.io/isort/docs/configuration/options.html#line-length))

This PR changes it to `110` which is what we had added to Airflow too ([reference](https://github.com/apache/airflow/blob/7efdeed5eccbf5cb709af40c8c66757e59c957ed/pyproject.toml#L18))

## Does this introduce a breaking change?
No

### Checklist
- [x] Created tests which fail without the change (if possible)
- [x] Extended the README / documentation, if necessary

### CODE REVIEW

**Comment by kaxil** [comment]
Note to reviewers:

- Review individual commits - https://github.com/astronomer/astro-sdk/pull/1030/commits/5c8fb434006a471dc4c088b8ac1d3d5c1e61bc34 and https://github.com/astronomer/astro-sdk/pull/1030/commits/8a8dd0e52c2e1654e5370f6aa033ffe851771615 . The last commit is re-generated

**Comment by kaxil** [comment]
https://github.com/astronomer/astro-sdk/pull/1030/commits/55674eac9f21ddba69bed5ff857dff03edbebe5b -- :( it happened again :/

**Comment by kaxil** [comment]
Ok found it, the main issue was fixed by adding the following to `.isort.cfg`:

```ini
known_first_party=astro,tests
```

File: sql-cli/conftest.py [hunk 1]
```diff
@@ -2,7 +2,6 @@
 from pathlib import Path
 
 import pytest
-
```
Comment (pankajkoti) [inline sql-cli/conftest.py:5]: Wondering if this blank line separator for local package should get deleted in this file and all files below for `sql-cli` directory. What do you think?

**Review by pankajkoti — APPROVED** [review]
Looks amazing to me, looks clean now ❤️ .

Just had couple of comments inline but realising now that they're unrelated to the scope of the PR. 
However, have one questions about the blank line separator getting removed for local `sql-cli` imports in `sql-cli` directory, but if that is correct, everything looks great to me.

File: python-sdk/example_dags/example_amazon_s3_postgres_load_and_save.py [hunk 2]
```diff
@@ -37,6 +38,4 @@ def example_amazon_s3_postgres_load_and_save():
     aql.cleanup()
 
 
-example_amazon_s3_postgres_load_and_save_dag = (
-    example_amazon_s3_postgres_load_and_save()
-)
+example_amazon_s3_postgres_load_and_save_dag = example_amazon_s3_postgres_load_and_save()
```
[the remark this answers is not in the document]
Comment (pankajkoti) [reply python-sdk/example_dags/example_amazon_s3_postgres_load_and_save.py:41]: We can ignore this.

File: python-sdk/example_dags/example_google_bigquery_gcs_load_and_save.py [hunk 3]
```diff
@@ -32,19 +33,17 @@
...
     @aql.dataframe(columns_names_capitalization="original")
     def extract_top_5_movies(input_df: pd.DataFrame):
         print(f"Total Number of records: {len(input_df)}")
-        top_5_movies = input_df.sort_values(by="rating", ascending=False)[
-            ["title", "rating", "genre1"]
-        ].head(5)
+        top_5_movies = input_df.sort_values(by="rating", ascending=False)[["title", "rating", "genre1"]].head(
+            5
+        )
         print(f"Top 5 Movies: {top_5_movies}")
```
[the remark this answers is not in the document]
Comment (pankajkoti) [reply python-sdk/example_dags/example_google_bigquery_gcs_load_and_save.py:47]: We can ignore this comment as it is outside the scope of the PR.

File: sql-cli/conftest.py (hunk 1 above)
Comment (kaxil) [reply sql-cli/conftest.py:5]: not sure -- it is auto-generated and doesn't affect runtime so 🤷

**Pull request merged**